packages/core/src/editor/Editor.tsx

```diff
@@ -28,7 +28,7 @@ export const Editor: React.FC<Partial<Options>> = ({
   useEffect(() => {
     if (context && options)
       context.actions.setOptions((editorOptions) => {
-        editorOptions = options;
+        Object.assign(editorOptions, options);
       });
   }, [context, options]);
```

Comment:
Actually, on a second look, there's more to this - the `options` object changes way too frequently, thus the `useEffect` hook will get triggered too frequently as well.

On top of my head, I can think of 2 options:
1. Use `lodash/isEqualWith` to compare the `options` prop with the previous `options`. But I'm not a fan of using deepEquals, so it would be great if we can avoid this.

2. Memoize the `options` object:
```tsx
export const Editor: React.FC<Partial<Options>> = ({
  children,
  onNodesChange,
  onRender,
  enabled,
  resolver,
  indicator,
  handlers,
}) => {
  const options = useMemo(
    () => ({
      ...
    }),
    [onNodesChange, onRender, enabled, resolver, indicator, handlers]
  );

  ...
});
```